Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -630,14 +630,16 @@ bool VolumeVerifier::CheckPartition(const Partition& partition)
     // IOS9 is the only IOS which can be assumed to exist in a working state on any Wii
     // regardless of what updates have been installed. At least Mario Party 8
     // (RM8E01, revision 2) uses IOS9 without having it in its update partition.
-    bool has_correct_ios = tmd.IsValid() && (tmd.GetIOSId() & 0xFF) == 9;
+    const u8 ios_ver = tmd.GetIOSId() & 0xFF;
```

Comment:
```suggestion
    const u64 ios_ver = tmd.GetIOSId() & 0xFF;
```

Otherwise this implicitly truncates to u8

Reply:
A u8 is all that's needed here, since the IOS slot number is limited to 8-bit. (It's the low 8 bits of the title ID.) I could remove the `& 0xFF`, since that's implied by the truncation.

Reply:
Right, but using `u64` here preserves the type, has no implicit truncations taking place, and works totally fine. If u8 is used, its preferable to use a static_cast to make the conversion explicit as far as the type system cares, otherwise this can cause warnings.